package.json

```diff
@@ -7,6 +7,10 @@
   "exports": {
     ".": "./src/index.js"
   },
+  "scripts": {
+    "format": "yarn prettier src *.md -w",
+    "lint": "yarn prettier src *.md -c"
```

Comment:
```suggestion
    "format": "yarn prettier . --write --ignore-unknown",
    "lint": "yarn prettier . --check --ignore-unknown"
```

Reply:
That means "format everything that prettier knows how to format"? I would like to exclude `package.json`, since it's managed by yarn.

Reply:
Oh well, I guess I can use `.prettierignore`

Reply:
yarn just does `JSON.stringify`, so they should agree